source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -397,9 +406,26 @@ export default class NetworkStatusStore extends Store {
         // Update localTimeDifference only in case NTP check status is not still pending
         if (localTimeInformation.status !== 'pending') {
           this.localTimeDifference = localTimeInformation.difference;
-          this.isNodeTimeCorrect =
+          const isNodeTimeCorrect =
             this.localTimeDifference != null && // If we receive 'null' it means NTP check failed
             this.localTimeDifference <= ALLOWED_TIME_DIFFERENCE;
+
+          if (
+            !isNodeTimeCorrect &&
+            this.isNodeTimeCorrect &&
```

Comment:
This can be confusable `! isNodeTimeCorrect  && this. isNodeTimeCorrect`. Maybe some more descriptive variable naming for localTimeDifference check should be better?

Reply:
@daniloprates let's discuss this once you start today 🙏

Reply:
@tomothespian @nikolaglumac I have changed it so it's easier to understand. Please, give it a look.